src/utils/screenUtils.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Checks if a given window size is considered as small.
+ * Returns `true`, if the windowSize.width <= 550
+ * @param windowSize { width?: number; height?: number }
+ * @returns {boolean}
+ */
+export const isSmallScreen = (windowSize: { width?: number; height?: number }): boolean => {
```

Comment:
height is not used

Reply:
Yes, I know. Moved it as is from echopediaWeb.

I thought about removing it, but I think this way it's more intuitive to use, since the user doesn't need to know if this is only depending on width, so he / she just can use it like this: (actual usage from echopediaWeb)

```ts
const windowSize = EchoUtils.Hooks.useWindowSize();

if(EchoUtils.Utils.isSmallScreen(windowSize)) {
  ...
}
```
Hmm, but based on this usage I think I could just move the `useWindowSize()` hook call inside `isSmallScreen()` to make it easier to use.

Reply:
As mentioned, converted it to a hook: useIsSmallScreen, which is based on the useWindowSize hook - so no params needs to be passed.